.github/workflows/pydpf-post.yml

```diff
@@ -1,23 +1,45 @@
 name: PyDPF-Post testing
 
 on:
+# Can be called by the CI
+  workflow_call:
+    inputs:
+      PyDPF-Post_branch:
+        default: ''
+        type: string
+      ANSYS_VERSION:
+        required: false
+        type: string
+        default: "231"
+    secrets:
+      DPF_PIPELINE:
+        required: true
+# Can be called manually
   workflow_dispatch:
     inputs:
       PyDPF-Post_branch:
-        description: 'PyDPF-Post branch to test'
+        description: 'PyDPF-Post branch or tag to test. Leave empty to get latest tagged version'
+        default: ''
+        type: string
+      ANSYS_VERSION:
+        description: "ANSYS version"
         required: true
-        default: 'master'
         type: string
+        default: "231"
```

Comment:
@PProfizi shouldn't we use a global variable for all the "231" references that might avoid to have a lot to change each release?

Reply:
@anslpa I do not think I can put an expression in the default value of workflow inputs.
The pipelines will have to be updated in a few places anyway.
Maybe NOT putting a default value is the solution, so that only the CI, and CI_release pipelines should be updated.